lib/dogapi/v1/comment.rb

```diff
@@ -9,51 +9,24 @@ class CommentService < Dogapi::APIService
 
       # Submit a comment.
       def comment(message, options = {})
-        begin
-          params = {
-            :api_key => @api_key,
-            :application_key => @application_key
-          }
-
-          body = {
-            'message' => message,
-          }.merge options
-
-          request(Net::HTTP::Post, "/api/#{API_VERSION}/comments", params, body, true)
-        rescue Exception => e
-          suppress_error_if_silent e
-        end
+        body = {
+          'message' => message,
+        }.merge options
+
+        request(Net::HTTP::Post, "/api/#{API_VERSION}/comments", nil, body, true)
       end
 
       # Update a comment.
       def update_comment(comment_id, options = {})
-        begin
-          params = {
-            :api_key => @api_key,
-            :application_key => @application_key
-          }
-
-          if options.empty?
-            raise "Must update something."
-          end
-
-          request(Net::HTTP::Put, "/api/#{API_VERSION}/comments/#{comment_id}", params, options, true)
-        rescue Exception => e
-          suppress_error_if_silent e
+        if options.empty?
+          raise "Must update something."
```

Comment:
Re-adding my comment here: this exception is not rescued anymore, I think we should remove it altogether for consistency